packages/fe-fpm-writer/test/unit/app.test.ts

```diff
@@ -14,7 +14,7 @@ type SAPUI5 = ManifestNamespace.JSONSchemaForSAPUI5Namespace;
  * @param settings.minVersion
  * @returns  Partial<Manifest>
  */
-function getTestManifest(settings?: { minVersion?: string }): Partial<Manifest> {
+function getTestManifest(settings?: { minVersion?: string | string[] }): Partial<Manifest> {
```

Comment:
By allowing the minVersion to be an array, we'll get a warning on lilne 34

`manifest['sap.ui5'].dependencies.minUI5Version = settings.minVersion;`

since the test `manifest` contains string.

Reply:
Not reproduceable. Please do pnpm install